gnucash/gnome/dialog-fincalc.c

```diff
@@ -445,8 +455,8 @@ fincalc_calc_clicked_cb(GtkButton *button, FinCalcDialog *fcd)
     for (i = 0; i < NUM_FIN_CALC_VALUES; i++)
     {
         text = gtk_entry_get_text(GTK_ENTRY(fcd->amounts[i]));
-        if ((text != NULL) && (*text != '\0'))
-            continue;
+        /* if ((text != NULL) && (*text != '\0')) */
+        /*     continue; */
         calc_value(fcd, i);
```

Comment:
Use `//` for single-line comments, but it's better to remove the code than to commit commented-out code.